### DISCUSSION TITLE
How can I specify the browser path?

### DISCUSSION THREAD

**Discussion by i18nsite** [webui-dev/webui#313] [Q&A] [answered]
How can I specify the browser path?
I want to integrate a browser into my project to avoid compatibility issues

**Comment by AlbertShown** [comment, marked as answer]
The earlier version had a custom browser option, but it was removed because there are too many things the user needs to provide to run a browser properly.

However, a very easy workaround:

1. create a window with `NoBrowser`
2. Get the window URL
3. Run a system command from your backend code to run the browser `.\MyBrowser\Browser.exe --url {URL}`
